doc/in-progress/move_lock_file.md

```diff
@@ -0,0 +1,11 @@
+# `Move.lock` file Move packages
```

Comment:
I wasn't sure what the "Move packages" bit at the end signifies here -- is it intentional?